nifi-nar-bundles/nifi-rules-action-handler-bundle/nifi-rules-action-handler-service/src/main/java/org/apache/nifi/rules/handlers/LogHandler.java

```diff
@@ -42,23 +42,26 @@
 public class LogHandler extends AbstractActionHandlerService {
 
     public static final PropertyDescriptor DEFAULT_LOG_LEVEL = new PropertyDescriptor.Builder()
-            .name("Log Level")
+            .name("logger-default-log-level")
+            .displayName("Log Level")
```

Comment:
Should this be Default Log Level? Also we might want to mention in the description that depending on the rules, the logging level may be set there and in that case this property is ignored.